Jenkinsfile

```diff
@@ -60,8 +59,11 @@ pipeline {
         stage('push image to private repo') {
             // when { branch 'main'}
```

Comment:
could we uncomment this line?
so would run nexus stage only in the main branch?